index.html

```diff
@@ -60,16 +61,38 @@ <h3>Average Grade <span class="badge badge-secondary average-grade"></span></h3>
                             <div class="input-group-prepend">
                                 <div class="input-group-text"><i class="fas fa-graduation-cap"></i></div>
                             </div>
-                            <input type="text" class="form-control" id="student-grade" name="grade" placeholder="Student Grade">
+                            <input type="number" class="form-control" id="student-grade" name="grade" placeholder="Student Grade">
+                        </div>
+                    </div>
+                    <input type="text" class="d-none" id="student-id" name="id" placeholder="Student ID">
+                    <div>
+                        <div class="d-inline form-button mr-2">
+                        </div>
+                        <div class="d-inline form-cancel">
                         </div>
                     </div>
-                    <button type="submit" class="btn btn-success mr-2">Add</button>
-                    <button type="reset" class="btn btn-outline-success">Cancel</button>
                 </form>
+                <!-- Modal HTML -->
+                <div id="myModal" class="modal fade error-message" tabindex="-1">

```

Comment:
Make sure to give your elements more meaningful `id`s in the future. Anything with `my` in it looks like you copied and pasted it from somewhere or that you are building a demo of some kind. A better `id` here would be something like `error-modal`.

Reply:
OK, thank you for the tip. Appreciate it.